Extensions/blacklist.js

```diff
@@ -943,8 +943,10 @@ XKit.extensions.blacklist = new Object({
 								tmp_word = tmp_word + " " + m_p_words[m_i + 6];
 							}
 
-							tmp_word = tmp_word.replace(/\,/g, '').replace(/\u2026/g, '');
-							tmp_word = tmp_word.replace(/[\.,-\/#!$%\^&\*;:{}=\-_`~()]/g, "").replace(/\s{2,}/g, " ");
+							const thisCodeIsBadAndTheFollowingEscapeIsGarbage = tmp_word;
```

Comment:
maybe a better name would be `unsanitised`

Reply:
done but with the 🇺🇸 spelling 😎